sass.py

```diff
@@ -107,8 +107,16 @@ def from_lambda(cls, name, lambda_):
         :rtype: :class:`SassFunction`
 
         """
-        argspec = inspect.getargspec(lambda_)
-        if argspec.varargs or argspec.keywords or argspec.defaults:
+        if PY3:
+            argspec = inspect.getfullargspec(lambda_)
+            varargs, varkw, defaults = (argspec.varargs, argspec.varkw,
```

Comment:
oh oops, I was in a hurry, that should only be in the python3 case (A newly added feature)

```python
def kwonly_func(x, *, y=None):
    """y may only be passed as  a kwarg"""
```